### DISCUSSION TITLE
Documentation is wrong

### DISCUSSION THREAD

**Discussion by mschipperheyn** [mercadopago/sdk-js#76] [General]
createCardToken should not be called with
expirationMonth
expirationYear

but with

cardExpirationMonth
cardExpirationYear

Guys, you should open source this or document and type this adequately. Open source is better. And open up an issue list. This discussion area is silly. 

Please hire me or some of the other pro devs here to help you guys get this sorted. I prefer to get paid to fix this stuff properly for everybody than to have to sort through compiled javascript to just fix for me.

**Comment by lucmantovani** [comment]
Hello @mschipperheyn!

Thank you for noticing and reporting the mistake!
In fact, it should accept both ways, so we will fix this as soon as possible and we'll let you know here when ready.

Also, if you want to improve the security of your checkout for both you and your customers, we recommend the usage of `fields` module.
By using it, card related fields are served inside iframes and thus cannot be accessed by anyone except us.

Here are some references for using this module:
- Technical Reference: https://github.com/mercadopago/sdk-js/blob/main/API/fields.md
- Implementation guide (Card Form): https://www.mercadopago.com.br/developers/en/docs/checkout-api/payment-methods/receiving-payment-by-card
- Implementation guide (Core Methods): https://www.mercadopago.com.br/developers/en/docs/checkout-api/payment-methods/receiving-payment-by-card-core-methods

As for the opportunities, you can always check our website for openings and apply to them: https://careers-meli.mercadolibre.com/pt
